Example Apps/Example ObjC/RemoteControlManager.m

```diff
@@ -0,0 +1,225 @@
+        self->_climateData = onInteriorVehicleData.moduleData.climateControlData;
+    }];
+}
+
+- (void)subscribeClimateControlData {
+    SDLGetInteriorVehicleData *getInteriorVehicleData = [[SDLGetInteriorVehicleData alloc] initAndSubscribeToModuleType:SDLModuleTypeClimate moduleId:_climateModuleId];
+    [_sdlManager sendRequest:getInteriorVehicleData withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    }];
+}
+
+- (void)turnOnAc {
+    NSDictionary<NSString *, id> *climateDictionary = @{@"acEnable": @YES};
+    SDLClimateControlData *climateControlData = [[SDLClimateControlData alloc] initWithDictionary:climateDictionary];
+    SDLModuleData *moduleData = [[SDLModuleData alloc] initWithClimateControlData:climateControlData];
+    SDLSetInteriorVehicleData *setInteriorVehicleData = [[SDLSetInteriorVehicleData alloc] initWithModuleData:moduleData];
+    
+    [self.sdlManager sendRequest:setInteriorVehicleData withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+        if(!response.success) { return; }
+    }];
+}
+
+- (void)turnOffAc {
+    NSDictionary<NSString *, id> *climateDictionary = @{@"acEnable": @NO};
+    SDLClimateControlData *climateControlData = [[SDLClimateControlData alloc] initWithDictionary:climateDictionary];
+    SDLModuleData *moduleData = [[SDLModuleData alloc] initWithClimateControlData:climateControlData];
+    SDLSetInteriorVehicleData *setInteriorVehicleData = [[SDLSetInteriorVehicleData alloc] initWithModuleData:moduleData];
+    
+    [self.sdlManager sendRequest:setInteriorVehicleData withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+        if(!response.success) { return; }
+    }];
+}
+
+- (void)setClimate {
+    SDLTemperature *temperature = [[SDLTemperature alloc] initWithFahrenheitValue:73];
+    NSDictionary<NSString *, id> *climateDictionary = @{@"acEnable": @YES, @"fanSpeed": @100, @"desiredTemperature": temperature, @"ventilationMode": SDLVentilationModeBoth };
+    
+    SDLClimateControlData *climateControlData = [[SDLClimateControlData alloc] initWithDictionary:climateDictionary];
+    SDLModuleData *moduleData = [[SDLModuleData alloc] initWithClimateControlData:climateControlData];
+    SDLSetInteriorVehicleData *setInteriorVehicleData = [[SDLSetInteriorVehicleData alloc] initWithModuleData:moduleData];
+    
+    [self.sdlManager sendRequest:setInteriorVehicleData withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+        if(!response.success) { return; }
+    }];
+}
+
+- (NSArray*)getButtons {
```

Comment:
Should be a readonly property with a getter. Make sure to specialize arrays if possible.